docs/basic-usage.md

```diff
@@ -0,0 +1,66 @@
+# Getting Set Up
+
+First, get the brooklyn-tosca project from github, and build the project
+
+    $ git clone git@github.com:cloudsoft/brooklyn-tosca.git
+    $ mvn clean install -DskipTests
+
+Then, so we can refer to it in future, store this location in a variable
+
+    $ cd brooklyn-tosca
+    $ export BROOKLYN_TOSCA=$(pwd)
+   
+## Running Alien4Cloud
+
+Now change to the directory that contains the built artifacts
+
+    $ cd $BROOKLYN_TOSCA/brooklyn-tosca-dist/target/brooklyn-tosca-dist-0.9.0-SNAPSHOT-dist/brooklyn-tosca-dist-0.9.0-SNAPSHOT/
```

Comment:
It may also be that they're not using tosca-dist + the standalone version of A4C.